src/services/db_interface.ts

```diff
@@ -232,18 +255,88 @@ export interface ArtifactModel {
   aid: string;
   /** Required. copr-build */
   type: string;
-  payload:
-    | PayloadBrewBuild
-    | PayloadKojiBuild
-    | PayloadDistGitPR
-    | PayloadRedHatModule
-    | PayloadFedoraModule
-    | PayloadProductMDCompose;
+  payload?: {};
   states: ArtifactState[];
   /** When the mongodb-document will be auto-removed, for example: scratch build */
   expire_at?: Date;
 }
 
+export type ArtifactTypeBrewBuild = 'brew-build';
+export type ArtifactTypeKojiBuild = 'koji-build';
+export type ArtifactTypeDistGitPR = 'dist-git-pr';
```

Comment:
The last letter should be lowercase according to the style guide.
```suggestion
export type ArtifactTypeDistGitPr = 'dist-git-pr';
```